packages/web-components/fast-components/src/accordion/accordion-item/accordion-item.ts

```diff
@@ -0,0 +1,34 @@
+import { attr, FASTElement, observable } from "@microsoft/fast-element";
+
+export enum AccordionHeadingLevel {
+    h1 = "1",
+    h2 = "2",
+    h3 = "3",
+    h4 = "4",
+    h5 = "5",
+    h6 = "6",
+}
+
+export class AccordionItem extends FASTElement {
+    @attr
+    public headinglevel: AccordionHeadingLevel = AccordionHeadingLevel.h2;
+
+    @attr({ mode: "boolean" })
+    public expanded: boolean = false;
+
+    @attr
+    public id: string;
+
+    @observable
+    public button: HTMLElement;
+
+    /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+    public clickHandler = (e: MouseEvent) => {
+        this.expanded = !this.expanded;
+        this.change();
+    };
+
+    private change = (): void => {
+        this.$emit("change", this.id);
```

Comment:
Does this need to pass `this.id` as data? The event will have the element accessible from the target property of the event so this seems redundant.

Reply:
Good to know, I was following a pattern I used elsewhere.